src/gui/folder.cpp

```diff
@@ -282,6 +282,39 @@ bool Folder::canSync() const
     return !syncPaused() && accountState()->isConnected();
 }
 
+bool Folder::dueToSync() const
+{
+    // conditions taken from previous folderman implementation
+    if (isSyncRunning() ||
+            etagJob()   ||
+            isBusy()    ||
+            !canSync()) {
+        return false;
+    }
+
+    ConfigFile cfg;
+    // the default poll time of 30 seconds as it had been in the client forever.
+    auto polltime = std::chrono::milliseconds(cfg.DefaultRemotePollInterval);
+
+    // ... and an user configured poll interval that might exist
+    auto polltimeCfg = cfg.remotePollInterval();
```

Comment:
There is no DEFAULT_REMOTE_POLL_INTERVAL any more, I removed the define. A wise man told me that "in modern C++ we don't use defines" ;-)

Reply:
But why do you export it :)
The function already handles the DefaultRemotePollInterval

Reply:
Yes, but I need to detect if the local configured time is different from the default time, because that would have precedence even over the time coming in from the capabilities. That is why I need it.

Reply:
Please pass `capabilities().remotePollInterval()` as a default to `cfg.remotePollInterval(capabilities().remotePollInterval())`
That way we can handle the default case directly in the config without the need to duplicate the minimal interval etc